_posts/2024/2024-01-31-why-go-to-a-rails-or-ruby-conference.md

```diff
@@ -0,0 +1,84 @@
+  source: "https://cliveandrews.com/"
+---
+
+Conferences have always been a big part of my career as a developer. Attending, speaking and running events—off the side of my desk—have played a massive part in my career, job prospects and personal relationships.
+
+Enough about me. What about you? Why should you take a day (or two, or three!) off work, spend a bunch of money, get lectured from a stage and then have to talk to strangers in a strange town halfway across the country?
+
+## Professional Development
+
+A conference is _not_ training. At most a talk can impart useful syntax, or explain a concept or help to bridge where you are to _where you could be_.
+
+A conference is a great place to find out the latest and greatest thinking in your particular area of work or to be introduced to facets of your career that you haven’t had the chance to consider in the everyday, unrelenting, demand for features and bug fixes.
+
+Even a couple of hours of workshop delivered as part of an event’s programming, _isn’t_ training. You do sometimes get full days of classroom training attached to conferences, but those tend to be specific, full-days & smaller groups; those _are_ training!
+
+If information isn’t being downloaded into your head, then why bother? The videos will be online soon and you can always read the documentation.
+
+Thinking like this misses the point. You won’t watch the videos, the YouTube counts under my own conference talks would indicate those numbers round to zero.
+
+You’re not enabling the serendipity of being in a talk you would have avoided where an insight unpacks a problem that your team have been struggling with for months.
+
+The relent from the day-to-day is a chance to pause and reflect on your everyday struggles and use the event as a springboard to better times.
+
+## Inspiration
+
+I think of two kinds of inspiration.
+
+One is the regular kind: a talk contains the nugget of understanding you need to take a new technique, or piece of thinking, day to your day job and solve a problem that you or your team have been facing.
+
+The other is more intangible, and easy to scoff about.
+
+Meeting someone, or seeing a talk, the changes the trajectory of your career. You might choose a product company, if you’ve been in agency life. You might lean hard into open source work and make that a foundation of the next phase of your career. You might meet someone who, in three years, starts a company and remembers that great meal you had and asks you to join them.
+
+All these things are possible if, and only if, you’re in the room where it happens.
+
+## Hiring
+
+Conferences are a great place to meet folks from other companies and many sponsors are actively trying to hire developers in the hallway track.
+
+Does that mean that if you send folks in your team they’re all going to leave for better places to work? No. If you are concerned that this may happen, then you have _a lot_ of work on your own organisation.
```

Comment:
Yup!!

Reply:
```suggestion
Does that mean that if you send folks in your team they’re all going to leave for better places to work? No. If you are concerned that this may happen, then you have _a lot_ of work to do on your own organisation.
```